src/control/lib/daos/pool_property.go

```diff
@@ -314,7 +314,7 @@ func PoolProperties() PoolPropertyMap {
 		"scrub": {
 			Property: PoolProperty{
 				Number:      PoolPropertyScrubMode,
-				Description: "Checksum scrubbing mode",
```

Comment:
I think that as long as the key names haven't changed, there shouldn't be any need to update ftests that are working with pool properties. May want to have @daltonbohning or @phender confirm, though.

Reply:
I'm mostly confident that is true. Looks like ftest uses the description only for logging purposes - not any verification or expected values of description